forge-gui/res/cardsfolder/upcoming/mizzix_replica_rider.txt

```diff
@@ -0,0 +1,16 @@
+﻿Name:Mizzix, Replica Rider
+ManaCost:4 R
+Types:Legendary Creature Goblin Wizard
+PT:4/5
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.wasNotCastFromYourHand | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigCopy | TriggerDescription$ Whenever you cast a spell from anywhere other than your hand, you may pay {1}{U/R}. If you do, copy that spell and you may choose new targets for the copy. If the copy is a permanent spell, it gains haste and "At the beginning of the end step, sacrifice this permanent."
+SVar:TrigCopy:AB$ CopySpellAbility | Cost$ 1 UR | Defined$ TriggeredSpellAbility | MayChooseTarget$ True | RememberNewCard$ True | SubAbility$ DBAnimate
+SVar:DBAnimate:DB$ Animate | ConditionDefined$ Remembered | ConditionPresent$ Permanent | Keywords$ Haste | Triggers$ EOTSac | Duration$ Permanent
```

Comment:
You're not defining what needs to be animated. Try Defined$ Remembered

Reply:
I'll give it a try to get it to work

Reply:
I got it to work, you mind me creating a new PR with the card ? 
![image](https://user-images.githubusercontent.com/67333662/207100896-c8aa37b6-a03d-4fc3-9bc9-008477f9ec4f.png)

Reply:
Feel free to create the PR, and thankee-sai!